website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -80,6 +80,51 @@ Use this variant only when you need to put interactive elements such as a button
 
 ![Example of toggle content](/assets/components/accordion/toggle-content-interactive.png =800x*)
 
+## Sizes
+
+Our accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium`, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, so accordionItem sizes are not interchangeable.
+
+![Image here of DONT change sizes - only card variant](/assets/components/accordion/accordion-size-Dont.png)
+!!!
+
+## Types
+
+Our accordion has two container types: `card` and `flush`.
+
+### Card
+
+The `card` variant is the default and recommended type, creating visual separation between accordion sections and other UI elements. It helps organize content blocks and distinguish between sections within tabs. Unlike `Cards`/`Tiles` or `Tables`, the `card` variant groups multiple expandable sections for flexible content management.
+
+!!! Do
+
+![image of DO example for the card variant](/assets/components/accordion/accordion-card-type-example.png)
+!!!
+
+### Flush
+
+Use the flush variant for tight spaces. For example, use it within other containers like cards and flyouts, or in sidebars where space is limited.
+
+!!! Do
+
+![image of DO example for the Flush variant](/assets/components/accordion/accordion-flush-example.png)
+!!!
+
+### Static AccordionItems
+
+The `isStatic` property allows an accordion item to be opened or closed by default, but not toggleable between its set `isOpen` property. This is useful when needing to represent a piece of information that is relevant to your accordion content, but highlighting the `isStatic` version as the exception within the group.
+
+An example of this would be the following:
+
+![terraform example of a run being triggered with a step that is skipped - use UI helper to show the skipped step as “skipped.”](/assets/components/accordion/accordion-isStatic-example.png)
+
+## AccordionItem
```

Comment:
This looks like an orphaned H2 with no content in it. Is this supposed to be the top level of content type and the rest below?

Reply:
Yeah, I intended this to be H2 for the content below. Do you feel like this structure feels a bit odd?

Reply:
In this scenario the subsequent heading should be an H3